signer/ctclient/ctclient.go

```diff
@@ -0,0 +1,64 @@
+// +build ctclient
+
+package ctclient
+
+import (
+	"bytes"
+	"encoding/asn1"
+	"encoding/binary"
+	"encoding/pem"
+
+	"crypto/x509"
```

Comment:
Ah; didn't realize that mattered to goimports. I think this whole PR might be moot if the build works on Mac now, but I'll wait for someone to confirm before closing.